estate/views/inherited_res_users_views.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```

Comment:
I forgot this one: you only need those two levels if you want to distinguish between update and noupdate data. Here, you don't need, so only `<odoo>` is enough, no need for `<data>`